lib/main.dart

```diff
@@ -128,4 +128,129 @@ class _HomePageState extends State<HomePage> {
       },
     );
   }
+
+  void routeToInfo(ObjectInfo picData) {
+    setState(() {
+      Navigator.push(
+        context,
+        MaterialPageRoute(builder: (context) => ArtworkInfoScreen(picData)),
+      );
+    });
+  }
+}
+
+class ArtworkInfoScreen extends StatelessWidget {
+  final ObjectInfo picData;
+
+  ArtworkInfoScreen(this.picData);
+
+  @override
+  Widget build(BuildContext context) {
+    final imageData = (picData.images == null) ? null : picData.images[0];
+    final artistData = (picData.people == null) ? null : picData.people[0];
+    return Scaffold(
+      backgroundColor: Colors.teal[100],
+      appBar: AppBar(
+        title: Text('Artwork Information'),
+        backgroundColor: Colors.teal[50],
+      ),
+      body: SafeArea(
+        child: Center(
+          child: Column(
+            children: <Widget>[
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: ListView(
+                    children: <Widget>[
+                      Hero(
+                        tag: picData.primaryImageUrl,
+                        child: Image.network(picData.primaryImageUrl),
+                      ),
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 15.0, top: 15.0),
+                        child: Wrap(
+                          alignment: WrapAlignment.center,
+                          children: [
+                            Text(
+                              picData.title + ' (${picData.date})',
+                              style: TextStyle(
+                                  fontSize: 20.0,
+                                  fontStyle: FontStyle.italic,
+                                  fontFamily: 'Raleway'),
+                            ),
+                          ],
+                        ),
+                      ),
+                      if (picData.people != null && artistData.role == 'Artist')
+                        Wrap(
+                          children: [
+                            Text(artistData.name + ' (${artistData.lifeSpan})',
+                                style: TextStyle(fontSize: 17)),
+                          ],
+                        ),
+                      if (picData.people != null)
```

Comment:
If this is a Travis thing, just disregard. I find this section a bit hard to read. Can you check the indentation?